VideoLocker/src/main/java/org/edx/mobile/user/UserAPI.java

```diff
@@ -68,7 +86,78 @@ public void deleteProfileImage(@NonNull String username) throws RetroHttpExcepti
         return userService.getBadges(username, page, ApiConstants.STANDARD_PAGE_SIZE);
     }
 
-    public List<EnrolledCoursesResponse> getUserEnrolledCourses(@NonNull String username) throws RetroHttpException {
-        return userService.getUserEnrolledCourses(username);
+    public List<EnrolledCoursesResponse> getUserEnrolledCourses(@NonNull String username, boolean tryCache) throws RetroHttpException {
+        List<EnrolledCoursesResponse> ret = null;
+        String json = null;
+
+        final String cacheKey = config.getApiHostURL() + "/api/mobile/v0.5/users/" + username + "/course_enrollments";
```

Comment:
@bguertin I'm not sure what you mean here.

In Api.java we see the line

String url = getBaseUrl() + "/api/mobile/v0.5/users/" + pref.getCurrentUserProfile().username
                + "/course_enrollments/";

and the implementation of getBaseUrl() is 

    public String getBaseUrl() {
        return config.getApiHostURL();
    }

so, I'd say that it already is just about the same as it could be.

Are you suggesting that I add a public getter method in Api.java? If so, I'd be calling into this deprecated class. Let me know if that is, indeed, what you want.